docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -6,66 +6,175 @@ aliases = ["/docs/grafana/latest/enterprise/license-restrictions"]
 weight = 300
 +++
 
-# License restrictions
+# Understanding Grafana Enterprise licensing
```

Comment:
Is it intentional that the name of this file be `license-restrictions.md`? How about `enterprise-licensing.md`?

Can you point me to the sheet and cell where this aligns with the info arch redesign plan?